File: programmers/약수의개수와덧셈.js
function findDivisor(num) {
    let count = 0
    for (let i = 1; i < num + 1; i ++) {
        if (num%i === 0) {
            count ++
        }
    }
    return count
}

function solution(left, right) {
    var answer = 0;
    for(let i = left; i < right + 1; i ++) {
        console.log(findDivisor(i))
        if (findDivisor(i)%2 === 0) {
            answer += i
        } else {
            answer -= i
        }
    }
    return answer;
}

File: sparta/약수의개수와덧셈.py
def countDivisor(divisor):
    count = 1 # 자기 자신
    for i in range(1, divisor // 2 + 1):
        if divisor % i == 0:
            count += 1
    return count

def solution(left, right):
    answer = 0
    for i in range(left, right + 1):
        if countDivisor(i) % 2 == 0:
            answer += i
        else:
            answer -= i
    return answer